cmd/cmdtest/testutils.go

```diff
@@ -377,6 +377,46 @@ func RunDockerCmdExec(args []string, t *testing.T) (string, error) {
 	return outBuffer.String(), err
 }
 
+// RunBuildahCmdExec runs the buildah command with the given args in a new process
+// The stdout and stderr are captured, printed, and returned
+// args will be passed to the buildah command
+// workingDir will be the directory the command runs in
+func RunBuildahCmdExec(args []string, t *testing.T) (string, error) {
```

Comment:
Just a suggestion, this cmd is very similar  to the docker one. We could add a parameter to `RunDockerCmdExec` for the command `buildah/docker` and rename it to `RunCmdExec`?